### DISCUSSION TITLE
An issue with codes in user queries

### DISCUSSION THREAD

**Discussion by danieltty** [sinaptik-ai/pandas-ai#819] [Q&A]
Let say I query pandasai to execute some operations on a database. It will work fine. But if I copy the exact code in its log and put it into the query instead of the original words, and ask it to run the codes line by line on the given database, it will not run correctly.
More precisely, it only runs the first line of code and presents the result, ignoring codes other than the first line.

**Comment by gventuri** [comment]
@tytung2020 thanks a lot for reaching out. Could you tell me more about the use case? Why are you trying to pass the pandas code to the llm asking to execute it instead of executing it?

**Comment by danieltty** [comment]
@gventuri hi thanks for reply. I am using it as a db query and manipulation tool. But I also want to allow more technical user to change the code in the middle, before sending it back to pandasai to work on the db.
My current way around it is by using chain of commands, running code lines one by one. Or simply run the code on the db. But this means no loops or any kind like that in the code.
